src/lang/pt-BR/Suites.json

```diff
@@ -1,6 +1,6 @@
 {
   "tableHeader": "Lista de Suites",
-  "tableSubHeader": "Esta é a lista de suites que este usuário possui permissoões para visualizar",
+  "tableSubHeader": "Esta é a lista de suites que tenho permissoões para visualizar",
```

Comment:
s/tenho/possuo (can't explain why... in this case the verb possuir/possess I guess sounds better)... but if you could fix my old typo as well, please haha. s/permissoões/permissões (extra o)

Reply:
Oh, BTW, just to clarify in case there is the potential for misinterpretation, I was not changing your own translation here (I do not know Brazilian Portuguese of course so to change your translation would be ridiculous), it is that I changed the text slightly in so that for the ``en-GB`` it reads 'I have' rather than 'the current user', which is more appropriate I think (probably the previous was for information during development). So I tried to change the ``pt-BR`` equivalent using Google translate, awaiting some help from a native speaker :smile:.

Thanks for help with the translations here, I'll add them in a new commit soon. Let me know if I missed anything once that goes up.